demo/src/index.tsx

```diff
@@ -17,10 +18,6 @@ import './main.css';
 // tslint:disable-next-line no-import-side-effect ordered-imports
 import '../../src/css/fancy-example.css';
 
-const Arrow = (): JSX.Element => (
```

Comment:
There are still arrows, but they're no longer explicit. Check out `fancy-example.css` and you'll see that they're now just a pseudo-element on `accordion__button`.